transpiler/src/parser/core/grammar/BitloopsParser.g4

```diff
@@ -837,15 +827,7 @@ serverDeclaration
     ;
 
 serverInstantiationOptions
-    // TODO Remove necessary comma on end of each line
-    : OpenBrace serverInstantiationOption* CloseBrace
-    ;
-
-serverInstantiationOption
-    : serverTypeOption    
-    | serverApiPrefixOption
-    | restServerPort 
-    | customServerOption
+    : OpenBrace (serverTypeOption | serverApiPrefixOption| restServerPort)* CloseBrace
```

Comment:
Do we need cors option now that customServerOption is gone? (there is no test for, it's used in the todo examples)